PYChen.md

```diff
@@ -1,3 +1,13 @@
+Last Week ([03/30 - 04/02)
+* [camera]
+	- planing demo goal and discuss implementation deatial, please refer to the link as below.
+	- https://taiwan.etherpad.mozilla.org/camera-new-features
```

Comment:
I read the pad but can't quite understand each item... Anyway, let me know if you need my input. Thanks.